src/App.tsx

```diff
@@ -1,9 +1,8 @@
-import React from "react";
-
 import { useState } from "react";
 import reactLogo from "./assets/react.svg";
 import viteLogo from "/vite.svg";
 import "./App.css";
+import { Button } from "./components";
```

Comment:
replace this to `import { Button } from "Components";`

We have added paths in tsconfig. This is to add absolute paths in our code.
Avoid relative paths.

Reply:
As we used vite, there is some dependency on vite.config.ts file for using alias. check it once @RushilBasappa